src/proxy_lib/proxy_lib.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+*/
+
+#ifndef UMF_PROXY_LIB_H
+#define UMF_PROXY_LIB_H 1
+
+#include <stddef.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifdef _WIN32
+#define WIN32_dllexport __declspec(dllexport)
```

Comment:
This is not needed since we use .def file

Reply:
OK. There is an error during linking on Windows. I thought it was needed to fix it, but it didn't help.
@igchor Do you know maybe how fix this error?
Removed.
Done.